chapters/parallel/index.md

```diff
@@ -0,0 +1,85 @@
+## Parallelization in Jina
+
+##### Learn about how Jina allows for parallelization of data processing
+
+### Feature description and expected outcome
+
+[comment]: <> (Outline)
+
+[comment]: <> (1. Parallel pods)
+
+[comment]: <> (2. Parallel peas within pod)
+
+[comment]: <> (2a. sharding)
+
+[comment]: <> (2b. parallel processing)
+
+Jina allows for parallelization of Peas and Pods. This generally leads to an increase in throughput, as the data is split across processes. However, depending on the configuration, this can also lead to 
```

Comment:
lead to...what?

Reply:
Forgot what I was going to write :thinking: Couldn't have been important then. Removed